go-apps/meep-rnis/sbi/rnis-sbi.go

```diff
@@ -175,18 +175,31 @@ func processActiveScenarioUpdate() {
 					mnc := ""
 					mcc := ""
 					cellId := ""
+					erabIdValid := false
 					if domain.CellularDomainConfig != nil {
 						mnc = domain.CellularDomainConfig.Mnc
 						mcc = domain.CellularDomainConfig.Mcc
+						cellId = domain.CellularDomainConfig.DefaultCellId
 					}
-					if poa.Poa4GConfig != nil {
-						if poa.Poa4GConfig.CellId != "" {
-							cellId = poa.Poa4GConfig.CellId
-						} else {
-							cellId = domain.CellularDomainConfig.DefaultCellId
+					switch poa.Type_ {
+					case mod.NodeTypePoa4G:
+						if poa.Poa4GConfig != nil {
+							if poa.Poa4GConfig.CellId != "" {
+								cellId = poa.Poa4GConfig.CellId
+							}
+						}
+						erabIdValid = true
+					case mod.NodeTypePoa5G:
+						if poa.Poa5GConfig != nil {
+							if poa.Poa5GConfig.CellId != "" {
+								cellId = poa.Poa5GConfig.CellId
+							}
 						}
+					case mod.NodeTypePoaWifi:
```

Comment:
Use "default:" for the unsupported technology types. This way we will not have to re-work this if we add for say mmWave

Reply:
added a default cases that is neither 4g-5g or wifi.. so will apply to new technology or POA-GENERIC

Reply:
- Remove `case PoaWifi` as it does the same as `default` + RNIS should only care about 3GPP tech.
- Add this comment in default: `// Empty cellId is ignored by RNIS`